openpype/pipeline/context_tools.py

```diff
@@ -43,7 +43,7 @@
 
 _is_installed = False
 _process_id = None
-_registered_root = {"_": ""}
+_registered_root = {"_": {}}
```

Comment:
This seems incorrect? It looks like `register_root` intends to register a `str` value. Why do we turn the default value into a `dict`?

Reply:
Because it should be dict since multiroot (from pype 2).

Reply:
Then `register_root` maybe get a clear docstring that `path` argument is actually `path_mapping` 

And might even make more sense changing the name even to `register_roots` or whatever - anyway, definitely source of confusion - maybe a docstring update there should suffice.

Reply:
The function is actually legacy and should disappear competelly, it exist only because of load functions that need new implementations. I don't have any time and energy to "make it proper" when it should in fact disappear. All I can do is to add some legacy warning into docstring, but not renaming.